reconcile/utils/terrascript_client.py

```diff
@@ -2900,10 +2945,14 @@ def unpack_namespace_info(namespace_info):
         return cluster, namespace
 
     @staticmethod
-    def get_dependencies(tf_resources):
+    def get_names_from_tf_resources(tf_resources: List[Resource]) -> List[str]:
```

Comment:
Minor point here, but it's a good practice to use more generic types for parameters if possible.  In this case, you should be able to use `Iterable[Resource]` which should work given that you're only looping through the results.

This is covered on the third bullet point [here](https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#type-hints).

The abstract types that are preferred can be found [here](https://docs.python.org/3/library/typing.html#typing.List).  Likewise, `(Mutable)Mapping` is preferred to `Dict` (again mostly for inputs).

Other than that the type usage looks good!

Reply:
Now using Iterable.

https://github.com/bkez322/qontract-reconcile/commit/55e6f94f04bcf98d3b2985a02cb6269c8b3666a7